test/ex_check/project_cases/formatter_issue_test.exs

```diff
@@ -21,7 +21,6 @@ defmodule ExCheck.ProjectCases.FormatterIssueTest do
 
     assert String.contains?(output, """
            ** (Mix) mix format failed due to --check-formatted.
-           The following files were not formatted:
```

Comment:
Can you explain the need for this change?

Reply:
OK figure out by myself (the wording differs among Elixir versions).

Reply:
Yes, in old Elixir versions it was `filed were not formatted`, now it is `files are not formatted`. So I just removed this part from a test.